homepage.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="homepage.css">
+    <script src="https://kit.fontawesome.com/928599f9b9.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header section">
+        <nav class="nav">
+            <div class="top-nav">
+                <i class="fa fa-facebook fa-nav" aria-hidden="true"></i>
+                <i class="fa fa-twitter fa-nav" aria-hidden="true"></i>
+                <a href="">English</a>
+                <a href="">My Page</a>
+                <a href="">Logout</a>
+            </div>
+            <div class="bottom-nav">
+                <a class="logo" href="">
+                    <img src="images/tdb.png" alt="logo image">
+                    <p class="logo-text">TD<br> Bliss</p>
+                </a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+                <ul class="links">
+                    <li class="link"><a href="index.html">Home</a></li>
+                    <li class="link"><a href="">Program</a></li>
+                    <li class="link"><a href="">Join</a></li>
+                    <li class="link"><a href="">Sponsor</a></li>
+                    <li class="link"><a href="">News</a></li>
+                    <li class="link last"><a class="logo" href="">Theatre Drama Bliss</a></li>
+                </ul>
+            </div>
+            
+        </nav>
+    </header>
+    <section class="headline section">
+        <p class="message">
+            "Passing on Traditions and Entertainment"
+        </p>
+        <h1 class="primary-text">
+            THEATRE DRAMA BLISS <br>
+            ANNUAL FESTIVAL <span class="year">2022</span>
+        </h1>
```

Comment:
- [ ] This content is too large and many of the other contents, kindly stick to the font size of the Behance

### Your design
![image](https://user-images.githubusercontent.com/86200356/203852742-b887b092-d45d-4a88-ab12-6599016e3231.png)

### Behance
![image](https://user-images.githubusercontent.com/86200356/203852529-90c746df-0faf-4ff7-a614-67d1e0c22d7d.png)